### ISSUE TITLE
Log which method of credential fetching succeeded

### ISSUE THREAD

**Issue by brettrowberry** [cognitect-labs/aws-api#217]
When I look through logs of my app starting up, I can see which credential fetch methods failed. I would love to also have a log message saying which method worked.

## Dependencies

``` clojure
{:deps 
  {com.cognitect.aws/api            {:mvn/version "0.8.524"}}
  {com.cognitect.aws:secretsmanager {:mvn/version "814.2.1008.0"}}
```

## Description with failing test case

What I see:

```text
Unable to fetch credentials from system properties.
Unable to fetch credentials from environment variables.
```

What I expect to see:

```text
Unable to fetch credentials from system properties.
Unable to fetch credentials from environment variables.
Fetch credentials from <method>.
```

## Stack traces

N/A

**Comment by dchelimsky** [comment]
Done in 7352ec647ff19f6a14cbbfc5e581c48a8183ebf1 h/t @MissInterpret . Will be in the next release.

**Comment by dchelimsky** [comment]
Released in [0.8.630](https://github.com/cognitect-labs/aws-api/blob/main/CHANGES.md#08630--2022-12-05)

**Comment by brettrowberry** [comment]
Thanks!
